invenio_accounts/config.py

```diff
@@ -206,3 +206,6 @@
 
 RECAPTCHA_PRIVATE_KEY = None
 """reCAPTCHA private key."""
+
+ACCOUNTS_INFO = True
```

Comment:
Now, that the tests are failing can I get you to change this to ``ACCOUNTS_USERINFO_HEADERS`` and set it to False by default.